pkg/localcontroller/manager/incrementallearningjob.go

```diff
@@ -192,7 +192,7 @@ func (im *IncrementalJobManager) evalTask(job *IncrementalLearningJob) error {
 	if currentType == sednav1.ILJobStageCondWaiting {
 		err := im.loadDeployModel(job)
 		if err != nil {
-			klog.Warningf("job(name=%s) failed to sync deploy model, and waiting it: %v",
+			return fmt.Errorf("job(name=%s) failed to sync deploy model, and waiting it: %v",
```

Comment:
it does not need job id info into `error` that leading to redundant message.

Reply:
ok! fix it. same as trainTask